src/api/schema.ts

```diff
@@ -216,6 +228,18 @@ export const lists = {
                 },
               },
             });
+
+            // add the application ID to the buyer entry.
+            await context.db.Buyer.updateOne({
+              where: { id: buyerId },
+              data: {
+                application: {
+                  connect: {
+                    id: applicationId,
+                  },
+                },
+              },
+            });
```

Comment:
Can we add unit test coverage for the new relationships please, in `keystone.test.ts`